WordPress/src/main/java/org/wordpress/android/ui/activitylog/detail/ActivityLogDetailFragment.kt

```diff
@@ -168,11 +188,27 @@ class ActivityLogDetailFragment : Fragment() {
             dialog.initialize(
                     it,
                     getString(R.string.activity_log_rewind_site),
-                    getString(R.string.activity_log_rewind_dialog_message, item.createdDate, item.createdTime),
+                    getString(
+                            R.string.activity_log_rewind_dialog_message,
+                            item.createdDate,
+                            item.createdTime
+                    ),
                     getString(R.string.activity_log_rewind_site),
                     getString(R.string.cancel)
             )
             dialog.show(requireFragmentManager(), it)
         }
     }
+
+    private fun buildTrackingSource() =
```

Comment:
1) Minor (🔍): I really liked the `forwardSlash` variable that you initially had, since this will make the whole string concatenation more explicit and readable. However, I would suggest having that as a constant instead, like `FORWARD_SLASH` (see full example below).
2) Minor (🔍): When a function is converted into an expression body, and there are braces then the equal can be on the same line if it fits. This will make the function look as a body function (see full example below).
3) Minor (🔍): Consider getting the string and checking if that is nullable instead of using `containKey(...)`. This will make it a bit more readable since the `SOURCE_TRACK_EVENT_PROPERTY_KEY` will only be utilised once (see full example below).

```
    private fun buildTrackingSource() = requireActivity().intent?.extras?.let {
        val source = it.getString(SOURCE_TRACK_EVENT_PROPERTY_KEY)
        when {
            source != null -> source + FORWARD_SLASH + DETAIL_TRACKING_SOURCE
            else -> DETAIL_TRACKING_SOURCE
        }
    }
```